apps/mark/frontend/src/app_root.tsx

```diff
@@ -293,29 +298,24 @@ export function AppRoot({
   const hasPrinterAttached = printerInfo !== undefined;
   const previousHasPrinterAttached = usePrevious(hasPrinterAttached);
 
-  const auth = useInsertedSmartcardAuth({
-    allowedUserRoles: [
-      'system_administrator',
-      'election_manager',
-      'poll_worker',
-      'cardless_voter',
-    ],
-    cardApi: card,
-    scope: {
-      // The BMD allows election managers to update the machine to use the election definition on
-      // the card in this case
-      allowElectionManagersToAccessMachinesConfiguredForOtherElections: true,
-      electionDefinition: optionalElectionDefinition,
-      precinct: appPrecinct,
-    },
-    logger,
-  });
-
-  const precinctId = isCardlessVoterAuth(auth)
-    ? auth.user.precinctId
+  const authStatusQuery = getAuthStatus.useQuery(electionHash);
+  const authStatus = authStatusQuery.isSuccess
+    ? authStatusQuery.data
+    : InsertedSmartCardAuth.DEFAULT_AUTH_STATUS;
```

Comment:
Any particular reason to use a default status vs just rendering `null` while loading?

Reply:
So we actually do render `null` while loading auth status:

https://github.com/votingworks/vxsuite/blob/2b546fb4fe48345b3301970b8d0205b7dec6a264/apps/mark/frontend/src/app_root.tsx#L644-L646

There are just a lot of pieces of data and hooks over the next few lines that use `authStatus`, and I didn't want to have to check if `authStatus` exists in all of them. And I can't move the `return null` up before the hooks because the number of hooks has to be consistent across renders.

Reply:
Gotcha, I didn't read closely, whoops. That makes sense. Hopefully the hooks will go away in time